apps/system/utils/netcmd_ping.c

```diff
@@ -313,9 +313,9 @@ int ping_process(int count, const char *taddr)
 	close(s);
 
 	printf("--- %s ping statistics ---\n", taddr);
-	printf("%d packets transmitted, %d received, %f\%% packet loss,\n", ping_send_counter,
+	printf("%d packets transmitted, %d received, %f\%% packet loss\n", ping_send_counter,
 		   g_ping_recv_counter,
-		   (100.0f * (float)(g_ping_recv_counter - ping_send_counter))/(float)g_ping_recv_counter);
+		   100.0f * (g_ping_recv_counter ? (float)(g_ping_recv_counter - ping_send_counter)/(float)g_ping_recv_counter : 1));
```

Comment:
I think that calculating the loss rate is wrong. it should be divide by g_ping_send_count not g_ping_recv_count.

Reply:
yes, I will modify